hackathon.html

```diff
@@ -135,6 +135,7 @@ <h2 id="dev-apis">dApp Developer APIs and Tools</h2>
                             <li><a href="https://developer.rchain.coop/rho-blockly">Rho Blockly</a>
                             <li><a href="https://developer.rchain.coop/rholang-playground">Rholang Playground</a>
                             <li><a href="https://github.com/rchain/rchain/blob/dev/rholang/reference_doc/rho_cheatsheet.pdf" >Rholang cheat-sheet</a></li>
+                            <li><a href="https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshareg">vs-code Live Share</a>
```

Comment:
404 error.
Link should be:
https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsls-vs